guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -189,21 +189,11 @@ Note that kickstart repositories should not be added to content views, as they a
 === Content Deployment
 
 Content deployment is the management of errata and packages on content hosts.
-There are two methods for content deployment on {Project}.
-The default method is *remote execution*, and the second method is the deprecated *Katello agent*.
 
 * *Remote execution* {endash} Remote execution allows installation, update, or removal of packages, the bootstrap of configuration management agents, and the trigger of Puppet runs.
```

Comment:
There only one item left. Maybe you can combine line 191 and 194 into one paragraph? Might need some minor rewording.